Sources/NIOFileSystem/FileSystem.swift

```diff
@@ -656,7 +656,7 @@ public struct FileSystem: Sendable, FileSystemProtocol {
 extension NIOSingletons {
     /// A suggestion of how many threads the global singleton ``FileSystem`` uses for blocking I/O.
     ///
-    /// The thread count is ``System/coreCount`` unless the environment variable
+    /// The thread count is the systems core count unless the environment variable
```

Comment:
Core count is slightly more complex than this as cgroup aware etc.

Reply:
Perhaps not super important to drop this distinction though.  Maybe available cores would be a better way of phrasing it.